packages/atomic/src/components/search/atomic-search-box-recent-queries/atomic-search-box-recent-queries.ts

```diff
@@ -1,45 +1,47 @@
 import {
   buildRecentQueriesList,
-  RecentQueriesList,
-  SearchBox,
+  type RecentQueriesList,
+  type SearchBox,
 } from '@coveo/headless';
-import {Component, Element, Prop, State, h} from '@stencil/core';
-import Clock from '../../../../images/clock.svg';
-import {SafeStorage, StorageItems} from '../../../../utils/local-storage-utils';
-import {once} from '../../../../utils/utils';
+import {LitElement, nothing} from 'lit';
+import {customElement, property, state} from 'lit/decorators.js';
+import {bindings} from '@/src/decorators/bindings';
+import type {LitElementWithError} from '@/src/decorators/types';
+import {SafeStorage, StorageItems} from '@/src/utils/local-storage-utils';
+import {once} from '@/src/utils/utils';
+import Clock from '../../../images/clock.svg';
 import {
   getPartialRecentQueryClearElement,
   getPartialRecentQueryElement,
-  RecentQueriesContainer,
-  RecentQueryClear,
-  RecentQueryIcon,
-  RecentQueryText,
-} from '../../../common/suggestions/stencil-recent-queries';
-import {
-  dispatchSearchBoxSuggestionsEvent,
-} from '../../../common/suggestions/suggestions-events';
+  renderRecentQuery,
+  renderRecentQueryClear,
+} from '../../common/suggestions/recent-queries';
+import {dispatchSearchBoxSuggestionsEvent} from '../../common/suggestions/suggestions-events';
 import type {
   SearchBoxSuggestionElement,
   SearchBoxSuggestions,
   SearchBoxSuggestionsBindings,
-} from '../../../common/suggestions/suggestions-types';
-import {Bindings} from '../../atomic-search-interface/atomic-search-interface';
+} from '../../common/suggestions/suggestions-types';
+import type {Bindings} from '../atomic-search-interface/interfaces';
 
 /**
  * The `atomic-search-box-recent-queries` component can be added as a child of an `atomic-search-box` component, allowing for the configuration of recent query suggestions.
  */
-@Component({
-  tag: 'atomic-search-box-recent-queries',
-  shadow: true,
-})
-export class AtomicSearchBoxRecentQueries {
-  private bindings!: SearchBoxSuggestionsBindings<SearchBox, Bindings>;
+@customElement('atomic-search-box-recent-queries')
+@bindings()
+export class AtomicSearchBoxRecentQueries
+  extends LitElement
+  implements LitElementWithError
+{
+  private suggestionsBindings!: SearchBoxSuggestionsBindings<
+    SearchBox,
+    Bindings
+  >;
   private recentQueriesList!: RecentQueriesList;
   private storage!: SafeStorage;
 
-  @Element() private host!: HTMLElement;
-
-  @State() public error!: Error;
+  @state() public error!: Error;
+  @state() bindings!: Bindings;
```

Comment:
do we really need bindings both from the search interface and from the search box?

if not (which I suspect) then we can remove this as well as the @bindings() decorator, and rename `suggestionsBindings` to `bindings`.